.flake8

```diff
@@ -29,6 +29,7 @@ ignore =
     TOR102,
 per-file-ignores =
     __init__.py: F401
+    torch/**: TOR901
```

Comment:
is TOR9 the torch.library thing? If so we want to limit it to `test/**`, not `torch/**`

Reply:
@zou3519 It's `per-file-ignores`, so it means ignore in `torch/**`, thus limit to `test/**` (as it only occurs in torch and test).

Reply:
ahh, thanks for pointing that out.